plugins/unmanaged_files/unmanaged_files_model.rb

```diff
@@ -73,6 +73,14 @@ def compare_with(other)
     comparison.push(common)
   end
 
+  def has_metadata?
+    @metadata ||= has_metadata || elements.any?(&:user)
+  end
+
+  def file_objects?
```

Comment:
I think this could be a more speaking name, maybe negate the value and call it `has_subdir_counts` or something like that? `file_objects` is just an internal name and doesn't really say what it means.